Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+#if canImport(UIKit)
+import UIKit
+
+public extension CustomerIOImplementation {
+    func enableAutoScreenviewTracking() {
+        let selector1 = #selector(UIViewController.viewDidAppear(_:))
```

Comment:
There are many `UIViewController.viewXXX` functions. Mind sharing more about why you chose `viewDidAppear`? Adding a code comment for that would be helpful for documentation.

Reply:
based this on how Segment handles theirs. Is there a lifecycle event that you think is more suited to tracking? https://github.com/segmentio/analytics-ios/blob/e35f1c984687f3216810145c8f1a8360fad20a72/Segment/Internal/NSViewController%2BSEGScreen.m#L77